xarchain/chain.json

```diff
@@ -0,0 +1,122 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "xarchain",
+  "chain_type": "cosmos",
+  "chain_id": "xarchain",
+  "website": "https://arcana.network/",
+  "pretty_name": "XarChain",
+  "status": "live",
+  "network_type": "testnet",
+  "bech32_prefix": "cosmos",
+  "daemon_name": "xarchaind",
+  "node_home": "$HOME/.xarchain",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "stxar",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0,
+        "high_gas_price": 0
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "stxar"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/arcana/xarchain",
+    "recommended_version": "v0.0.5-test-rc",
+    "compatible_versions": [ ],
```

Comment:
I believe compatible_versions must have values (can repeat the recommended_version), and the versions.json file must agree (which is missing the compatible_versions property).